pipelines/controller/src/main/java/com/google/fhir/analytics/metrics/Stats.java

```diff
@@ -42,33 +38,19 @@ public class Stats {
    * weightage for each step is a rough approximation derived based on the time taken for that step
    * as compared to the other steps.
    *
-   * @param metricQueryResults
+   * @param cumulativeMetrics
    * @return Stats
    */
-  public static Stats createStats(@Nullable MetricQueryResults metricQueryResults) {
+  public static Stats createStats(@Nullable CumulativeMetrics cumulativeMetrics) {
 
-    if (metricQueryResults == null) {
+    if (cumulativeMetrics == null) {
       return null;
     }
 
     Stats stats = new Stats();
-    long totalNoOfResources = 0l;
-    for (MetricResult<GaugeResult> gauge : metricQueryResults.getGauges()) {
-      if (gauge.getName().getName().startsWith(MetricsConstants.TOTAL_NO_OF_RESOURCES)) {
```

Comment:
So it seems these `TOTAL_NO_OF_RESOURCES` metrics are not needed anymore and I wonder do we still need the  `DoFn` in `FhirEtl` that were just to register these metrics? Those are at line 309 of `FhirEtl.java` above (of the old code, or line 320 of the new one). It would be great if we can remove those `ParDo`s (if you remember, we had some discussions around those when you originally added them).

Reply:
Yes, now the `TOTAL_NO_OF_RESOURCES` are not needed for tracking the progress bar. However, these metrics which gets reported are being in the UI. Probably, we will have to report these if needs to be shown in the UI. Currently, thinking of ways to report this.

Reply:
I see, but given that those stats are somewhat broken now (because of #839), I would suggest to remove those `ParDo`s in this PR and implement a better way of reporting total numbers as part of fixing #839.

Reply:
Ok, removed the ParDo function now, will address it as part of https://github.com/google/fhir-data-pipes/issues/839